modules/cli/src/main/scala/scala/cli/exportCmd/MavenProjectDescriptor.scala

```diff
@@ -134,7 +124,7 @@ final case class MavenProjectDescriptor(
         testDeps: ShadowingSeq[Positioned[AnyDependency]],
         isCompileOnly: Boolean
       ): Seq[MavenLibraryDependency] = {
-        val scopePriorities       = List()
+        List()
```

Comment:
```suggestion
```
Looks like remove unused only removes lhs?

Reply:
...yeah, that's `scalafix` at work